docs/guide/intro/index.adoc

```diff
@@ -13,3 +15,129 @@ implement new WidFly subsystems and expose new server capabilities.
+For example:
+```
+<configuration>
+...
+ <plugin-options>
+  <jboss-fork-embedded>true</jboss-fork-embedded>
+ </plugin-options>
+...
+</configuration>
+```
+
+* For the link:https://github.com/wildfly-extras/prospero[prospero] tool, you can define a `provisioning.xml` file that contains the options.
+A `provisioning.xml` file example:
+```
+<installation xmlns="urn:jboss:galleon:provisioning:3.0">
+    <feature-pack location="wildfly@maven(org.jboss.universe:community-universe):current"/>
+    <options>
+        <option name="jboss-fork-embedded" value="false"/>
+        <option name="jboss-maven-dist"/>
+    </options>
+</installation>
+```
+
+#### Installation Plugin options
+
+[cols="1,2,3,4"]
+
+|===
+|Option 
+|Type
+|Default value
+|Description 
+
+|jboss-bulk-resolve-artifacts
+|Boolean
+|false
+|Maven artifacts that are needed for the WildFly server installation are resolved in a bulk operation instead of 
+being resolved individually. That can speed-up the provisioning time.
+
+|jboss-dump-config-scripts
+|String
+|NONE
+|A path to a directory in which WildFly CLI script files used to generate the server configurations are stored.
+
+|jboss-fork-embedded
+|Boolean
+|false
+|Fork in a separate process the generation of the server configuration files. Server configuration generation implies the usage of a WildFly Embedded server.
+It is advised to set this env variable to true, specially when having multiple executions of the provisioning inside the 
+same process (e.g.: multiple executions of Galleon\|Wildfly\|Bootable JAR Maven plugins).
+
+|jboss-maven-dist
+|Boolean
+|false
+|Provision a thin WildFly server installation with JBoss Modules modules jar resolved from Maven local cache 
+and not copied to the `modules` directory.
+
+|jboss-maven-repo
+|String
+|NONE
+|A path to a directory in which all the resolved Maven artifacts jars and associated pom files are stored. 
+The generated directory complies with the filesystem layout of a Maven repository. This respository can then be used as the local cache 
+of a thin WildFly server (see `jboss-maven-dist` option). 
+
+|jboss-overridden-artifacts
+|'\|' separated list of maven coordinates in the following syntax: GroupId:ArtifactId:Version:[Classifier:]Extension
+|NONE
+|A list of Maven artifact coordinates used to override the artifact versions that are defined in the provisioned WildFly feature-pack.
+For example, to override undertow-core: `io.undertow:undertow-core:2.3.1.Final::jar`
+
+|jboss-reset-embedded-system-properties
+|Comma separated list of String
+|NONE
+|When executing a forked process to generate the server configuration, all system properties defined in the parent process are inherited by the forked process.
+In case you want to reset WildFly specific system properties, set this property to an empty value. 
+To erase additional specific system properties, set this property to a comma separated list of system property names to erase.
```

Comment:
does that mean that, if the system property is set via `jboss-reset-embedded-system-properties`, both Wildfly properties and the provided property will be removed?

Reply:
Yes, made it clear.